.travis.yml

```diff
@@ -0,0 +1,36 @@
+language : python
+cache: pip
+
+sudo: true
+
+addons:
+    chrome: stable
+
+services:
+  - docker
+
+python:
+  - "3.6"
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install node
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+install:
+  - pip install webviz_components webviz_plotly
+  - pip install -r requirements.txt
+  - pip install -r dev-requirements.txt
+  - pip install -r tests/requirements.txt
```

Comment:
Remove this line, and instead put content of `tests/requirements.txt` into `dev-requirements.txt`?